efficientdet/det_model_fn.py

```diff
@@ -613,15 +612,15 @@ def scaffold_fn():
           save_steps=1000, output_dir=params['model_dir'], show_memory=True)
       training_hooks.append(profile_hook)
 
-      # Report memory allocation if OOM
-      class OomReportingHook(tf.estimator.SessionRunHook):
+    # Report memory allocation if OOM
+    class OomReportingHook(tf.estimator.SessionRunHook):
 
-        def before_run(self, run_context):
-          return tf.estimator.SessionRunArgs(
-              fetches=[],
-              options=tf.RunOptions(report_tensor_allocations_upon_oom=True))
+      def before_run(self, run_context):
+        return tf.estimator.SessionRunArgs(
+            fetches=[],
+            options=tf.RunOptions(report_tensor_allocations_upon_oom=True))
```

Comment:
Does oom  tracker will decrease training speed?

Reply:
It shouldn't, as it  is triggered only if OOM happens. Did you have this experience before?

Reply:
I speed up training process by remove profilehook and oomhook.
I'm not care about each hook's performance.

Reply:
You are right, this hook indeed slow down the training, I have disabled it in default.